### PR TITLE
[feat] Creation of Home Page & General ReactApp setup

### PR DESCRIPTION
**Pull request by Hamdrive**
As per the requirement of the project, it has now been converted to a fully fledged react application.

Features/Changes implemented in this PR:
1. Setup basic routing for the ecommerce app using React Router
2. Converted the Home Page into full fledged React components
3. Implementation of CSS modules to handle specificity with styles

As of now the React app is properly functioning as a desktop app, but also has some features of mobile app.
Lateron it will also be hosted on Netlify

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,42 +1,21 @@
-import "./App.css";
-import logo from "./logo.png";
+import {Routes, Route} from "react-router-dom"
+import Cart from "./pages/cart/Cart.jsx"
+import Wishlist from "./pages/wishlist/Wishlist.jsx"
+import Signup from "./pages/signup/Signup.jsx"
+import Login from "./pages/login/Login.jsx"
+import Products from "./pages/products/Products.jsx";
+import Home from "./pages/home/Home";
```
Comment (junaidshaikh-js) [inline src/App.js]: Create index file in pages folder and export all the pages from the index file and then import all the pages from the single index file here.

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import { Link } from "react-router-dom";
+import "../styles/footer.css"
+
+export function Footer() {
+  return (
+    <div class="py-md footer footer-doc">
```
Comment (junaidshaikh-js) [inline src/components/Footer.jsx]: In React we use  `className` instead of `class`

File: src/pages/home/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
+// import "./home.css";
```
Comment (junaidshaikh-js) [inline src/pages/home/Home.jsx]: code comment should be removed

**Review by junaidshaikh-js — COMMENTED** [review]
Suggested changes can be done. 

Good Work!

File: src/styles/navbar.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+/* Navbar icon badges styles */
+.top-right-icon {
+  top: 0;
+  right: 0;
+  transform: translate(25%, -50%);
+}
+
+/* Media queries for tablets and up */
+@media screen and (max-width: 768px) {
```
Comment (saadahmadhere) [inline src/styles/navbar.css:53]: use of relative units is good practice.

**Review by saadahmadhere — COMMENTED** [review]
Looks great! Use of variables has made the code even cleaner. Well structured repo. Good suggestions were made by the other reviewers too, need to check on that.

**Comment by Hamdrive** [comment]
Owing to the major changes made which make this pull request obsolete, I have closed the PR and am thankful to @saadahmadhere and @junaidshaikh-js for taking the time to review my PR.

File: src/App.js (hunk 1 above)
Comment (Hamdrive) [reply src/App.js]: Have import/export all the components from index file, good pointer!

File: src/components/Footer.jsx (hunk 2 above)
Comment (Hamdrive) [reply src/components/Footer.jsx]: Yes I have not refactored my code from HTML to JSX, have done so now, thanks!

File: src/pages/home/Home.jsx (hunk 3 above)
Comment (Hamdrive) [reply src/pages/home/Home.jsx]: code comments cleaned up

File: src/styles/navbar.css (hunk 4 above)
Comment (Hamdrive) [reply src/styles/navbar.css:53]: That is true but I would prefer to use pixels here just so it is easier to know which screen size I am targeting directly, and also to me makes this media query easy to read

**Pull request closed without merging**